cmd/deploy/command_configuration.go

```diff
@@ -134,7 +129,14 @@ func addEnvVars(ctx context.Context, cwd string) error {
 			if err != nil {
 				return err
 			}
-			repo = repoHTTPS.String()
+			if repoHTTPS != nil {
+				repo = repoHTTPS.String()
+			} else {
+				// ir repo was parsed but hasnt got a valid schema
+				// fallback to empty repository
+				oktetoLog.Warning("retrieved local repository")
```

Comment:
I wouldn't show a warning to the user, What do you think about logging it into the logs files with an info level?